tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -237,6 +237,7 @@
             </class>
             <class name="org.ballerinalang.test.balo.record.OpenRecordTypeReferenceTest" />
             <class name="org.ballerinalang.test.balo.record.ClosedRecordTypeReferenceTest" />
+            <class name="org.ballerinalang.test.balo.object.ReadOnlyObjectBaloTest"/>
```

Comment:
No need to add as a separate entry. This is already enabled at https://github.com/ballerina-platform/ballerina-lang/pull/26824/files#diff-3835b6074874bfd7bcb2e191247640239cbc9c4546822113a142766502ade096R131